logger/logger_writer.go

```diff
@@ -146,6 +147,8 @@ func queueMsg(lvl Level, prefix, format string, v ...interface{}) (err error) {
 		return
 	}
 
+	msg.time = time.Now()
+
```

Comment:
Ya, reset() is better in freeMsg()
defer issue is just cosmetic, but it's confusing.

Not sure about bufio.Write, looks like it's returning errors, we'll possibly append to original failed msg.
https://github.com/golang/go/blob/master/src/bufio/bufio.go#L593